digitalocean/database/resource_database_cluster.go

```diff
@@ -609,8 +613,12 @@ func flattenMaintWindowOpts(opts godo.DatabaseMaintenanceWindow) []map[string]in
 func setDatabaseConnectionInfo(database *godo.Database, d *schema.ResourceData) error {
 	if database.Connection != nil {
 		d.Set("host", database.Connection.Host)
-		d.Set("port", database.Connection.Port)
-		d.Set("uri", database.Connection.URI)
+		if database.EngineSlug == kafkaDBEngineSlug {
+			// default for kafka will be Public SASL port, consistent with UI
+			d.Set("port", kafkaPublicSSLPort)
```

Comment:
Are the ports guaranteed to be stable?  Maybe we should grab it from the response? Something like:

```go
const kafkaSASL = "sasl"
if database.Connection.ApplicationPorts != nil {
	if saslPort, ok := database.Connection.ApplicationPorts[kafkaSASL]; ok {
		d.Set("port", saslPort)
	}
}
```